.github/workflows/ci-matrix.yml

```diff
@@ -11,7 +11,6 @@ jobs:
           - beta
           - nightly
         features:
-          - surf-client
```

Comment:
Instead of adding `async-std` further down, shouldn't you add it here instead of `surf-client`?